code/game/machinery/cryopod.dm

```diff
@@ -317,8 +317,7 @@
 		if(istype(W,/obj/item/device/pda))
 			var/obj/item/device/pda/P = W
 			if(P.id)
```

Comment:
don't need this `if` check, becuase that's what `QDEL_NULL` is for to begin with!